app/questionnaire/questionnaire_schema.py

```diff
@@ -87,6 +87,9 @@ def get_show_on_hub_for_section(self, section_id):
     def get_summary_for_section(self, section_id: str) -> Mapping:
         return self._sections_by_id.get(section_id).get("summary", {})
 
+    def get_summary_title_for_section(self, section_id: str):
```

Comment:
`secttin.py` has a method called `get_page_title` which returns `get_title_for_section` but I think it makes sense to return `get_summary_title_for_section` first.
```python
    def get_page_title(self):
        return self._schema.get_summary_title_for_section(
            self._section_id
        ) or self._schema.get_title_for_section(self._section_id)
```

Reply:
Thats a good spot